src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -78,7 +78,7 @@ std::string jsonRequestTreat
     if ((answer = parseDataP->ent.res.check(ciP, EntitiesRequest)) != "OK")
     {
       OrionError error(SccBadRequest, answer);
-      return error.render(ciP, "");
+      return setStatusCodeAndSmartRender(ciP, error);
```

Comment:
I was thinkin in that option... but at the end given that the logic on setStatusCodeAndSmartRender() deals with OrionError *and* ConnectionInfo, I opted for using a function external to OrionError.

However, it is not a strong position. I can try to move setStatusCodeAndSmartRender() to a OrionError class method is you think is smarter in that way.